### DISCUSSION TITLE
Run GraphQL Hive with Docker Compose on GCP VM and expose to the Internet

### DISCUSSION THREAD

**Discussion by stanislavhordiyenko** [graphql-hive/console#3915] [Help] [answered]
Hi,

I've configured a Linux VM in GCP, installed Docker, Docker Compose. Downloaded the latest community yaml file, commented volume configuration (as it has permission issues and stack couldn't be run, so no volumes) for each service, ran the stack, however when I try to access it with public IP address, too much redirects are happening, and an empty black page is loaded.

What should be done in the configuration to make it work? I changed localhost to 0.0.0.0 as well, with no luck.

Please, advise and let me know if you need more information.

**Comment by kamilkisiela** [comment, marked as answer]
It's not enough information for me to help you narrow down the issue.
I don't know how you access anything there, I don't know what page you're trying to access, you did not share any logs from the containers.

You can try to see our `development` directory as a reference.
